test/functional/test_framework/messages.py

```diff
@@ -244,27 +244,28 @@ class CInv:
         MSG_TX | MSG_WITNESS_FLAG: "WitnessTx",
         MSG_BLOCK | MSG_WITNESS_FLAG: "WitnessBlock",
         MSG_FILTERED_BLOCK: "filtered Block",
-        4: "CompactBlock",
-        5: "WTX",
+        MSG_CMPCT_BLOCK: "CompactBlock",
+        MSG_WTX: "WTX",
+        0xffffffff: "Error",
```

Comment:
I'd prefer to report anything that is unkonwn as "Error" instead of specifically adding `0xffffffff`  here.

Reply:
This was only needed for the test. I tried a few [different tests](https://github.com/bitcoin/bitcoin/commit/6c0f4aefe373bb89188e91464326dabf0f9f3397]. [This one](https://github.com/bitcoin/bitcoin/commit/1ab179b78d63c7e96af7dbad6caf3efb82afa01a) doesn't require adding entries to `typemap`. I don't think any of the tests cover the code change, only the test framework change.

Reply:
I'm not sure what to do here. This was only needed for the test. I tried a few [different tests](https://github.com/bitcoin/bitcoin/commit/6c0f4aefe373bb89188e91464326dabf0f9f3397); the first one doesn't require adding entries to `typemap`. I don't think any of the tests cover the CInv::type code change, only the test framework change.

Reply:
Dropped this line.